packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -75,6 +75,11 @@ export interface AutocompleteProps<ItemType>
    * @default false
    */
   clearAfterSelect?: boolean;
+  /**
+   * If this is set to `true` the dropdown menu will stay open after the select
+   * @default false
```

Comment:
true or false? :)

Reply:
sorry, default will be true, will update the text